samples/sample_common/src/sample_utils.cpp

```diff
@@ -71,7 +71,8 @@ mfxStatus CopyBitstream2(mfxBitstream *dest, mfxBitstream *src)
 CSmplYUVReader::CSmplYUVReader()
 {
     m_bInited = false;
-    m_ColorFormat = MFX_FOURCC_YV12;
+    m_ColorFormat 
```

Comment:
I think no need to change this line

Reply:
Why?

Reply:
Because it introduces unnecessary delta without valuable improvement